Gemfile.lock

```diff
@@ -258,7 +258,7 @@ GEM
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0.5.3)
     erubis (2.7.0)
-    eventmachine (1.0.8)
+    eventmachine (1.2.3)
```

Comment:
Then, we need to find the right version of guard-rails. The one that depends on an eventmachine version that works.